components/admin_real/desktop/Calendar.jsx

```diff
@@ -21,6 +21,9 @@ const dateCellRender = (cellDate, data) => {
       {filteredReservations.map((item) => (
         <li key={item.id}>
           <ReservationBadge item={item} />
+          <span>{item.time}</span>
```

Comment:
this feels like it could be its own component - maybe we will reuse this somewhere in the future

Reply:
yeah we definitely need to make it its own. I am working on this now, so there will be chagnes